### PR TITLE
fixes  + testSetup

### PR DESCRIPTION
**Pull request by jakobbraun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,5 +99,27 @@ Start with the [mapping definition example](doc/gettingStartedWithSchemaMappingL
 
 # Logging & Debugging
 
+The following links explain logging and debugging for the Virtual Schema in general:
+
 * [Logging for Virtual Schemas](https://github.com/exasol/virtual-schemas/blob/master/doc/development/remote_logging.md)
 * [Remote debugging Virtual Schemas](https://github.com/exasol/virtual-schemas/blob/master/doc/development/remote_debugging.md)
+
+For logging and debugging in the test code, things are easier, 
+as most of the setup code is already includes in the AbstractExasolTestInterface.
```
Comment (morazow) [inline README.md:108]:
```suggestion
as most of the setup code is already included in the AbstractExasolTestInterface.
```

File: src/main/java/com/exasol/adapter/dynamodb/mapping/OverflowException.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * Exception thrown if the size of the string from DynamoDB is longer than the configured size.
+ */
+public class OverflowException extends ColumnValueExtractorException {
+    private static final long serialVersionUID = 3544045333431038655L;
+
+    /**
+     * Create an instance of {@link OverflowException}.
+     * 
+     * @param message message
+     * @param column  column that cased the exception
```
Comment (morazow) [inline src/main/java/com/exasol/adapter/dynamodb/mapping/OverflowException.java:13]:
```suggestion
     * @param column  column that caused the exception
```

File: src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnValueExtractor.java [hunk 3]
```diff
@@ -18,11 +20,23 @@
...
     }
 
     @Override
     protected ValueExpression mapValue(final DocumentNode<DocumentVisitorType> documentValue) {
-        return StringLiteral.of(mapJsonValue(documentValue));
+        final String jsonValue = mapJsonValue(documentValue);
+        if (jsonValue.length() > this.column.getMaxLength()) {
+            if (this.column.getOverflowBehaviour().equals(ToJsonPropertyToColumnMapping.OverflowBehaviour.EXCEPTION)) {
+                throw new OverflowException(
+                        "The generated JSON did exceed the configured maximum size. You can either increase the column size if this column or set the overflow behaviour to NULL.",
```
Comment (morazow) [inline src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnValueExtractor.java:32]:
```suggestion
                        "The generated JSON did exceed the configured maximum size. You can either increase the column size of this column or set the overflow behaviour to NULL.",
```

File: src/test/java/com/exasol/adapter/dynamodb/OpenLibrary.java [hunk 4]
```diff
@@ -131,13 +134,23 @@ public void flush() {
                 return;
             }
             final TableWriteItems writeRequest = new TableWriteItems(this.tableName).withItemsToPut(this.batch);
-            final BatchWriteItemOutcome batchWriteItemOutcome = this.dynamoClient.batchWriteItem(writeRequest);
-            LOGGER.info("# Unprocessed items: " + batchWriteItemOutcome.getUnprocessedItems().size());
+            try {
+                final BatchWriteItemOutcome batchWriteItemOutcome = this.dynamoClient.batchWriteItem(writeRequest);
+
```
Comment (morazow) [inline src/test/java/com/exasol/adapter/dynamodb/OpenLibrary.java:139]:
```suggestion
```

**Comment by morazow** [comment]
Addresses minor typos.

Because ToJsonColumnMapping class has 4-5 parameters at the moment, it would make sense to create a builder for it with next additional parameter.

**Comment by jakobbraun** [comment]
> Because ToJsonColumnMapping class has 4-5 parameters at the moment, it would make sense to create a builder for it with next additional parameter.

ok

**Pull request merged**